### PR TITLE
Update first chapter with new content

### PR DESCRIPTION
**Pull request by novafurry**
This pull request updates the first chapter of the project with new content. It includes the following changes:

- Created a new file called "your-first-page.md" which contains the initial version of the first page.

- Deleted the old file "writing-text.md" as it has been replaced by "your-first-page.md".

- Added a new image called "explaing.jpg" which explains HTML markup.

- Deleted two images called "macos-install-python.png" and "windows-install-python.png".

- Updated the "manifest.json" file to include the new "your-first-page.md" file in the lessons section.

The new content in "your-first-page.md" provides a basic HTML structure and explains the purpose of each HTML tag. It also includes examples and images to illustrate the concepts.

### CODE REVIEW

File: include/browser.svg
Comment (infinity-atom) [inline include/browser.svg:1]: Could this be put into the markdown inline?

File: internal/setup.md [hunk 1]
```diff
@@ -2,73 +2,116 @@
...
+however it is suggested to use the VS Code Desktop app.
+<details>
+    <summary><b>Install VS Code as a Desktop App</b></summary>
+    Pick your operating system:
     <details>
-        <summary>Install VS Code as a Desktop App</summary>
-        Pick your operating system:
-        <details>
-            <summary>Windows</summary>
-            <ol>
```
Comment (infinity-atom) [inline internal/setup.md:11]: You can use markdown lists in a markdown file

File: internal/setup.md [hunk 2]
```diff
@@ -2,73 +2,116 @@
...
-                <li>Type <pre>bash </pre> and hit <kbd>enter</kbd> <div class="alert alert-primary">It's ok if nothing happens, this step is just to ensure everything will work correctly</div></li>
-                <li>Type <pre>cd ~/Downloads </pre> and hit <kbd>enter</kbd></li>
-                <li>Now type <pre>sudo apt install -y ./code</pre> and hit <kbd>tab</kbd>, it should autofill the name of the file you just downloaded, if not, enter the name of the downloaded file, ensuring it matches exactly, including the <pre>./</pre> preceeding.</li>
-                <li>now, hit enter and type your password when asked</li>
-                <li>You can now find Visual Studio Code in your app menu.</li>
-            </ol>
-        </details>
-        <details>
-            <summary>Arch Linux</summary>
-            You're an arch user, so i'm just gonna cut to the chase as I trust you know what you're doing
```
Comment (infinity-atom) [inline internal/setup.md:42]: I think you're missing instructions for arch

**Review by infinity-atom — APPROVED** [review]

File: include/browser.svg
Comment (novafurry) [reply include/browser.svg:1]: i know i replied to this on discord- but for publicity's sake, replying here again

no, it goes all screwy, for whatever reason

File: internal/setup.md (hunk 2 above)
Comment (novafurry) [reply internal/setup.md:42]: arch users know what they're doing
if you don't know how, you probably shouldn't use arch...

**Pull request merged**